Documents/idea document 2/ideadocument2.md

```diff
@@ -10,5 +10,16 @@
 # Speficications & Execution Plan
 
 # Expected Outcomes
+As mentioned earlier in the objective, the goal of this application is to identify and characterise neighborhoods in Amsterdam. In the following mockup of the user interface(UI) it will become visible how this could look.
```

Comment:
I think a comma should be added after (UI) so:
In the following mockup of the user interface (UI), it will become visible how this could look.